technical-committee/shared-development-roadmap/epics/shared/fair-tool-discovery.md

```diff
@@ -0,0 +1,220 @@
+---
+#(This frontmatter block follows YAML syntax)
+id: fair-tool-discovery #epic-title-in-lowercase-with-hyphens
+coordinator: Maarten van Gompel #person or list of people
+wp: [ 1, 2, 3, 4, 5, 6] #involved workpackages
+github-projects-link: https://github.com/orgs/CLARIAH/projects/1 #link to a specific project under here
+participants:
+    - Maarten van Gompel (KNAW HuC)
+themes: [ Metadata, DevOps, Curation, Vocabularies, Sustainability, Processing, Search ]
+evaluation: #overall evaluation for the (best) implementation of this epic
+    trl: 0 #technology readiness level
+    cl: 0 #compliance level
+    srl: 0 #stakeholder readiness level
+---
+
+## FAIR Tool Discovery
+
+### Rationale
+
+> Describe, from a high-level perspective, the rationale of what important role
+> the service(s) that implement this epic fulfill in the CLARIAH infrastructure
```

Comment:
Usually, with templates on GitHub, the placeholder text is _replaced_ by the actual text. Did you keep the placeholders on purpose. If so, why?

Reply:
I'm automatically removing the placeholder text upon generating the final PDF (via pandoc, latex and some scripting), so then the burden is not on the users. I also figured it might offer some extra clues/context for interpretation whilst we are still in the editing stage.